### ISSUE TITLE
Alternative preprocessing code with xarray 

### ISSUE THREAD

**Issue by xuezhengllnl** [big-data-lab-umbc/multivariate-weather-data-clustering#17]
This is the code to preprocess, including normalizing, the input data:

> import numpy as np
import xarray as xr
from sklearn import preprocessing
from sklearn.preprocessing import MinMaxScaler
min_max_scaler = preprocessing.MinMaxScaler() # calling the function

> var_list =['sst','sp','u10','v10','slhf','sshf','t2m'] 

> ecmwf_hourly=xr.open_dataset('./ERA5_meteo_sfc_2021_daily.nc')
null_value = -1.0
ivar = 0
for var_id in var_list:
    var0 = ecmwf_hourly[var_id].values
    var1d=min_max_scaler.fit_transform(var0.reshape(var0.shape[0],-1))
    var1d[np.isnan(var1d)]= null_value 
    if(ivar ==0):
        input1d = var1d
    else:
        input1d = np.concatenate((input1d,var1d),axis=1) 
    ivar += 1
